docs/source/tutorials/otx_cli/how_to_train/detection.rst

```diff
@@ -168,35 +176,21 @@ For example, to decrease batsch size to 4, fix the number of epochs to 100 and d
 5. The training results are ``weights.pth`` and ``label_schema.json`` files located in ``outputs`` folder, training logs can be found in the ``outputs/logs`` dir.
 
 .. code-block::
-
-  2022-12-29 00:59:51,961 - mmdet - INFO - workflow: [('train', 1)], max: 200 epochs
-  [ INFO ] workflow: [('train', 1)], max: 200 epochs
-  [ INFO ]  workflow: %s, max: %d epochs
-  2022-12-29 00:59:51,965 | INFO : cancel hook is initialized
-  2022-12-29 00:59:51,965 | INFO : logger in the runner is replaced to the MPA logger
-  2022-12-29 00:59:51,975 | INFO : Update Lr patience: 3
-  2022-12-29 00:59:51,975 | INFO : Update Validation Interval: 2
-  2022-12-29 00:59:51,975 | INFO : Update Early-Stop patience: 5
-  2022-12-29 01:00:30,180 | INFO : Epoch [1][1/32]        lr: 1.333e-03, eta: 282 days, 22:46:42, time: 38.204, data_time: 0.462, memory: 4669, current_iters: 0, loss_cls: 1.1113, loss_bbox: 0.5567, loss_centerness: 0.5920, loss: 2.2600, grad_norm: 3.6441
+  ...
+  2023-01-10 05:40:21,520 | INFO : Update Lr patience: 3
+  2023-01-10 05:40:21,520 | INFO : Update Validation Interval: 2
+  2023-01-10 05:40:21,520 | INFO : Update Early-Stop patience: 5
+  2023-01-10 05:40:23,140 | INFO : Epoch [1][1/31]        lr: 1.333e-03, eta: 11 days, 14:44:47, time: 1.619, data_time: 0.961, memory: 4673, current_iters: 0, loss_cls: 1.1261, loss_bbox: 0.6514, loss_centerness: 0.6337, loss: 2.4112, grad_norm: 18.5789
 
   ...
-  ---------------iou_thr: 0.5---------------
-
-  +-----------+-----+------+--------+-------+
-  | class     | gts | dets | recall | ap    |
-  +-----------+-----+------+--------+-------+
-  | Platelets | 76  | 310  | 1.000  | 0.897 |
-  | RBC       | 819 | 4070 | 0.994  | 0.903 |
-  | WBC       | 72  | 516  | 1.000  | 0.988 |
-  +-----------+-----+------+--------+-------+
-  | mAP       |     |      |        | 0.929 |
-  +-----------+-----+------+--------+-------+
-  2022-12-29 01:08:52,434 | INFO : run task done.
-  2022-12-29 01:08:53,010 | INFO : Adjusting the confidence threshold
-  2022-12-29 01:08:53,520 | INFO : Setting confidence threshold to 0.32500000000000007 based on results
-  2022-12-29 01:08:53,521 | INFO : Final model performance: Performance(score: 0.8315842078960519, dashboard: (17 metric groups))
-
-Now we have the Pytorch object detection model trained with OTX, that we can use for evaliation, export, optimization and deployment. 
+  2023-01-10 05:52:33,985 | INFO : run task done.
+  2023-01-10 05:52:35,682 | INFO : Inference completed
+  2023-01-10 05:52:35,683 | INFO : called evaluate()
+  2023-01-10 05:52:35,907 | INFO : F-measure after evaluation: 0.5487693710118504
+  2023-01-10 05:52:35,907 | INFO : Evaluation completed
+  Performance(score: 0.5487693710118504, dashboard: (1 metric groups))
+
+After about 15 minutes of training we have the PyTorch object detection model trained with OTX, that we can use for evaliation, export, optimization and deployment. 
```

Comment:
This time may very depending on the hardware. Let's avoid this. This is also related to POT time estimation

Reply:
I think, it's good to let user know, what to expect on this dataset. I agree, that it depends highly on the hardware, but I mentioned used hardware at the beginning of the page. 
Maybe, the compromise here will be to highlight that this characteristic depends on hardware, but keep this value for GeForce 3090.